content/02.introduction.md

```diff
@@ -76,17 +77,62 @@ For SARS-CoV-2 it is still being investigated whether, in addition to being spre
 
 Among known coronaviruses, SARS-CoV-2 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi:10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253].
 This suggests that SARS-CoV-2 may have originated in viral reservoirs of similar hosts; however, current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].
 
-### COVID-19: Mechanisms and Presentation
-
-#### Immunological Mechanisms of Coronavirus-driven Disease in Humans
-
-Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
+The viral spike protein, _S_, plays a critical role in viral replication.
+The N-terminal S1 domain used for binding and the C-terminal S2 domain for fusion [@doi:10.3390/v4061011].
+The S1 domain evolves more rapidly than the S2 domain, and different coronaviruses bind to difference receptors.
+Betacoronaviruses are known to bind to CEACAM1, Neu 5,9 Ac2, and ACE2 [@doi:10.3390/v4061011].
+
+### COVID-19: Infection and Presentation
+
+#### Pathogenic Mechanisms of Coronaviruses
+
+Until more information becomes available about the pathogenesis of SARS-CoV-2 specifically, insight can be gained by examining how the human body responds to viral pathogens generally, how infection from related viruses proceeds, and how this information intersects with what is known about COVID-19.
+Following exposure to a pathogen such as a virus, pathogenesis involves three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}]. 
+The human immune response is double-pronged, with the innate and the adaptive immune response.
+Through innate immunity, also known as humoral or antibody-mediated immunity, the body can eliminate pathogens in circulation that have not yet entered cells.
+Adaptive immunity, or the cell-mediated response, is the process by which infected cells are destroyed.
+The antibody-mediated response works to prevent or reduce entry into cells, while the cell-mediated response works to stop the pathogen's replication and spread.
+
+##### The Innate Immune Response
+
+The antibody-mediated immune response is innate and provides a way through which the host can detect and respond to a pathogen in circulation.
+This response depends on several types of leukocytes (white blood cells) that are professional phagocytic cells, which means they are able to detect and engulf an antigen, enveloping it in a phagosome.
```

Comment:
(reminder note to get this: line 99 & line 100 categorizations seem mixed up, will get back to this later/soon)

Reply:
(still reminder notes but maybe it'll be useful to clarify when you're splitting the immunity types/sub-sections: there's categorizations innate immunity & adaptive immunity [and maybe more], there's also another type of categorization which is sort of similar or parallel of nonspecific immune defenses & specific immune defenses, and I do not know if there's a preference in terminology; within specific immune defenses & the similar term adaptive immunity there's commonly a terminology splitting into humoral/antibody/b-cell/~extracellularAntigen-sort-of based immunity & the cell-based/cytotoxic-T-cell-stuff/~intracellularAntigen-sort-of based immunity) & T-helper cells are related to both (though there's also there some categorizations of TH1 and TH2); from what I remember these are the typical categorizations used, and there's also big terms like inflammation & there's also the complexity and interactions between big terms which I do not remember or have not read enough about. From a brief look, it seems like you have both specific things & general things, and it seems you may need to double-check your references or word-choice for the general things comparing it to what I remember while for the specific things I haven't taken the time yet to read what you wrote carefully though I see another review has :-)   . It's good you're identifying what seems relevant/important; meanwhile, since I'm not very comfortable with immunology (even if I do some more reading or review I don't have enough experience to feel comfortable with it unless I'm checking something specific) I can't give you exact comments.)

Reply:
In this subsection draft, there's a mixture of good information but some of it is innate and some of if is acquired. Sometimes there are interactions between innate and acquired & sometimes some cells can work as either/both innate and acquired, but still it might be simpler to separate these better so there's less confusion when reading. We can figure out ways to re-assort the information between innate & adaptive...